### ISSUE TITLE
Black Myth WuKong on Mac has error: please use the epic launcher to start the game

### ISSUE THREAD

**Issue by arelouse** [Heroic-Games-Launcher/HeroicGamesLauncher#3982]
### Describe the bug

I installed the latest version of Heroic and tried it with Crossover and Whisky on my Apple M1 Pro. 
I could see the WUKONG icon on the Dock and it showed the error"Please use the Epic Launcher to start the game. Connection to the internet is required." Other Wine options like GPTK had the same problem.

### Add logs

```text
Launching "Black Myth: Wukong" (legendary)
Native? No
Installed in: /Users/xxx/Games/Heroic/BlackMythWukong

System Info:
CPU: 10x Apple M1 Pro
Memory: 34.36 GB (used: 14.94 GB)
GPUs:

OS:  14.5.0 (darwin)

The current system is not a Steam Deck
We are not running inside a Flatpak container

Software Versions:
  Heroic: 2.15.1 Emeth
  Legendary: 0.20.35 Lowlife (Heroic)
  gogdl: 1.1.1
  comet: comet 0.1.2

  Nile: 1.1.1 Will A. Zeppeli

Game Settings: {
	"autoInstallDxvk": true,
	"autoInstallDxvkNvapi": false,
	"autoInstallVkd3d": false,
	"preferSystemLibs": false,
	"enableEsync": true,
	"enableMsync": true,
	"enableFsync": false,
	"nvidiaPrime": false,
	"enviromentOptions": [],
	"wrapperOptions": [],
	"savesPath": "/Users/xxxx/Games/Heroic/BlackMythWukong/b1/Saved/SaveGames/xxxxx",
	"showFps": false,
	"useGameMode": false,
	"battlEyeRuntime": false,
	"eacRuntime": false,
	"language": "",
	"beforeLaunchScriptPath": "",
	"afterLaunchScriptPath": "",
	"wineVersion": {
		"bin": "/Users/xxxx/Downloads/CrossOver.app/Contents/SharedSupport/CrossOver/bin/wine",
		"name": "CrossOver - 24.0.4",
		"type": "crossover",
		"wineserver": "/Users/xxxx/Downloads/CrossOver.app/Contents/SharedSupport/CrossOver/bin/wineserver"
	},
	"wineCrossoverBottle": "ABC",
	"winePrefix": "/Users/xxxx/Games/Heroic/Prefixes/default/Black Myth Wukong",
	"ignoreGameUpdates": false
}

Game launched at: Tue Aug 27 2024 20:25:47 GMT+0800 (China Standard Time)

Winetricks packages: 

Launch Command: HEROIC_APP_NAME=f53c5471fd0e47619e72b6d21a527abe HEROIC_APP_RUNNER=legendary GAMEID=umu-0 HEROIC_APP_SOURCE=epic STORE=egs LD_PRELOAD= CX_BOTTLE=ABC WINE_FULLSCREEN_FSR=0 WINEESYNC=1 WINEMSYNC=1 /Applications/Heroic.app/Contents/Resources/app.asar.unpacked/build/bin/arm64/darwin/legendary launch f53c5471fd0e47619e72b6d21a527abe --wine /Users/xxxx/Downloads/CrossOver.app/Contents/SharedSupport/CrossOver/bin/wine --language zh_Hans

Game Log:
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Using CrossOver Bottle "ABC"
[cli] INFO: Launching f53c5471fd0e47619e72b6d21a527abe...
msync: bootstrapped mach port on wine-f3c0850-msync.
msync: up and running.
============= End of log =============
```

### Steps to reproduce

1. Go to 'library'
2. Click on 'Black Myth WuKong'
3. Wait
4. See error "Please use the Epic Launcher to start the game. Connection to the internet is required."

### Expected behavior

I can start the game and play it.

### Heroic Version

Latest Stable

### System Information

System Info:
CPU: 10x Apple M1 Pro
Memory: 34.36 GB (used: 14.94 GB)
GPUs:

OS:  14.5.0 (darwin)

**Comment by DeeDeeRanged** [comment]
Had the same issue but it resolved itself. My take on this is it was an issue with the authentication servers of epicgames as I even had trouble just logging in with a browser. After a few hours everthing was normal again.

**Comment by imLinguin** [comment]
The error was fixed in Heroic 2.15.2
